pkg/security/ebpf/c/include/hooks/exec.h

```diff
@@ -726,6 +726,9 @@ int __attribute__((always_inline)) send_exec_event(ctx_t *ctx) {
     // send the entry to maintain userspace cache
     send_event_ptr(ctx, EVENT_EXEC, event);
 
+    // as previously registered memory will become unreachable, we'll have to unregister the TLS
+    unregister_span_memory();
+
```

Comment:
👋🏻 Have you double checked we can still identify the original span for an RCE ? 🤔 
I think this was the original rational not to remove it here ...

Reply:
I _**think**_ it's ok though

Reply:
yeah let's do the ebpf at least in another PR, will be easier to revert if needed and will be easier to track as a potential change

Reply:
Ok, let me split it into another one for this change